doc/RELEASE-NOTES-0.2.0.md

```diff
@@ -0,0 +1,137 @@
+# RELEASE NOTES for "Good Peers" sprint
+
+The 3rd sprint of our development focused on the following issues:
+* ready all objects in the system for network exchange: define canonical form and serialization
+* build sync infrastructure
+* further work on internal state structure: Authenticated Dictionaries
+* further research on consensus architecture
+
+The packages for the this release are available at the following location:
+https://github.com/aeternity/epoch/releases/tag/v0.2.0-good-peers
+
+To fully appreciate the newest developments you need to experience syncing nodes and sharing blocks.
+
+The easiest way to achieve it, is to deploy 3 local nodes and see how they communicate.
+Please find detailed information about custom configuration here:
+https://github.com/aeternity/epoch/blob/4d1baf36207dfeff2d2f1a5256d37285574b7457/doc/CUSTOMIZATION.md
```

Comment:
we can shorten the commit part, e.g.
`https://github.com/aeternity/epoch/blob/4d1baf3/doc/CUSTOMIZATION.md`

Reply:
Addressed in https://github.com/aeternity/epoch/pull/266/commits/24a8d225f2cef992ca7bd6ecf3a37dcbaa3ca80a